app/usecases/stats.py

```diff
@@ -10,31 +10,6 @@
 from app.models.stats import Stats
 
 
-class StatsInfo(NamedTuple):
-    user_id: int
-    mode: Mode
-
-
-STATS: dict[StatsInfo, Stats] = {}
-
-
-async def fetch(user_id: int, mode: Mode) -> Optional[Stats]:
-    stats_info = StatsInfo(
-        user_id=user_id,
-        mode=mode,
-    )
-
-    if stats := STATS.get(stats_info):
-        return stats
-
-    stats = await fetch_db(user_id, mode)
-    if not stats:
-        return None
-
-    STATS[stats_info] = stats
-    return stats
-
-
 async def fetch_db(user_id: int, mode: Mode) -> Optional[Stats]:
```

Comment:
since we don't have `fetch` anymore, `fetch_db` should be renamed to just `fetch`, because generic-ism like i mentioned